### PR TITLE
enh: enable skipLibCheck in tsconfig

### PR DESCRIPTION
**Pull request by HaecheonLee**
This enables skipLibCheck in `tsconfig`.

- This resolves errors generated when running `tsc`.

### CODE REVIEW

File: core/tsconfig.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "outDir": "dist",
     // MikroORM
     "experimentalDecorators": true,
+    "skipLibCheck": true,
```
Comment (exAspArk) [inline core/tsconfig.json:11]: 👍 

Would you be able to remove the `--skipLibCheck` flag in `build.yml` and `Makefile` then?

Comment (HaecheonLee) [reply core/tsconfig.json:11]: I don't see the flag in those files. Can you check again?

Comment (exAspArk) [reply core/tsconfig.json:11]: Sorry, I added them in the `main` branch recently, so you'd need to update this branch to see them

https://github.com/BemiHQ/bemi/blob/c0d6e7d78fef79192c2add3807722b743ba4f72e/Makefile#L4

Comment (HaecheonLee) [reply core/tsconfig.json]: Yes, my bad that my branch was based on my fork reap. It's updated now.

Comment (exAspArk) [reply core/tsconfig.json]: Sweet, thank you!

**Pull request merged**